custom_dc/quickstart.md

```diff
@@ -33,7 +33,7 @@ The "services" Docker container consists of the following Data Commons component
 
 ## Prerequisites
 
-- Obtain a [GCP](https://console.cloud.google.com/welcome) billing account and project.
+- Obtain a [GCP](https://cloud.google.com/docs/get-started) billing account and project.
```

Comment:
Looks like this has changes from the other PR. Best to submit the other one, sync and then submit this one.

Reply:
Done.